### DISCUSSION TITLE
Connection refused tcp4000

### DISCUSSION THREAD

**Discussion by tontonchow** [teslamate-org/teslamate#2400] [Q&A] [answered]
Hi

Since last update, i'm unable to connect with http on my 4000 port (ERR_CONNECTION_REFUSED), update from 1.25 to 1.26, last time i tried to udpate to 1.25.2 i have to rollback on my raspberry

Docker-compose ps reply the service is UP on both 3000 and 4000 port

     Name                   Command               State                    Ports
--------------------------------------------------------------------------------------------------
pi_database_1    docker-entrypoint.sh postgres    Up      5432/tcp
pi_grafana_1     /run.sh                          Up      0.0.0.0:3000->3000/tcp,:::3000->3000/tcp
pi_mosquitto_1   /docker-entrypoint.sh /usr ...   Up      0.0.0.0:1883->1883/tcp,:::1883->1883/tcp
pi_teslamate_1   tini -- /bin/sh /entrypoin ...   Up      0.0.0.0:4000->4000/tcp,:::4000->4000/tcp

il my logs i found error but no information about this 4000TCP rejected 

Attaching to pi_teslamate_1
teslamate_1  | waiting for postgres at database:5432
teslamate_1  | sleep: cannot read realtime clock: Operation not permitted
teslamate_1  | (no logger present) unexpected logger message: {log,error,"~s~n",["erts_poll_wait() failed: eperm (1)\n"],#{error_logger=>#{emulator=>true,tag=>error},time=>1643386306160882}}
teslamate_1  | (no logger present) unexpected logger message: {log,error,"~s~n",["erts_poll_wait() failed: eperm (1)\n"],#{error_logger=>#{emulator=>true,tag=>error},time=>1643386306160931}}

Someone could help me with this issue ?

**Comment by cwanja** [comment, marked as answer]
You still need to apply the fixes per #2302. Or upgrade your Raspberry Pi to Bullseye. See the release notes in 1.26 - https://github.com/adriankumpf/teslamate/releases/tag/v1.26.0

> Bump Docker app base image to Debian 11
> Raspberry Pi users unfortunately have to upgrade to Raspbian Bullseye or install the backports version libseccomp2 (see #2302)

**Reply by tontonchow** [reply]
Great, as simple as you say, thanks for this quick reply

**Comment by tontonchow** [comment]
sorry i've missed this information I'll try to follow up
